dev-vcs/tortoisehg/tortoisehg-5.3.1.ebuild

```diff
@@ -33,26 +33,7 @@ DEPEND="${RDEPEND}
 	doc? ( >=dev-python/sphinx-1.0.3 )"
 
 # Workaround race condition in build_qt
-DISTUTILS_IN_SOURCE_BUILD=1
-
-python_prepare_all() {
-	if [[ ${L10N+set} ]]; then
-		cd i18n/tortoisehg || die
-		local x y keep
-		for x in *.po; do
-			keep=false
-			for y in ${L10N}; do
-				if [[ ${y} == ${x%.po}* ]]; then
-					keep=true
-					break
-				fi
-			done
-			${keep} || rm "${x}" || die
-		done
-		cd "${S}" || die
-	fi
-	distutils-r1_python_prepare_all
-}
+#DISTUTILS_IN_SOURCE_BUILD=1
```

Comment:
Why are you leaving comments?

Reply:
Because I don't ever thought that I'll send my changes to Gentoo.